crates/bench/benches/uv.rs

```diff
@@ -152,20 +152,20 @@ mod resolver {
         let options = OptionsBuilder::new().exclude_newer(exclude_newer).build();
 
         let build_context = BuildDispatch::new(
-            client,
-            &cache,
-            interpreter,
-            &index_locations,
-            &flat_index,
-            &index,
-            &git,
-            &in_flight,
+            client.clone(),
+            cache,
+            interpreter.clone(),
+            index_locations,
+            flat_index.clone(),
+            index.clone(),
+            git,
+            in_flight,
```

Comment:
[auto-claim when?](https://smallcultfollowing.com/babysteps/blog/2024/06/21/claim-auto-and-otherwise/)